common-practices-tools/security/README.md

```diff
@@ -167,16 +167,15 @@ Finally, there is no good reason not to be making backups: a one _terabyte_ exte
 
 ### Securely delete files and wipe disks
 
-When you delete a file, it doesn't actually go away. Usually, all that occurs is the file name and a pointer to its bits are removed from a directory listing. With the proper tools, deleted files can be recovered. For this reason, it's important that old disks be securely wiped before being given or thrown away. Some pointers are given below:
+When you delete a file, it doesn't actually go away. Usually, all that occurs is the file name and a pointer to its bits are removed from a directory listing. With the proper tools, deleted files can be recovered. For this reason, it's important that old disks be securely wiped or permanently destroyed before being given away, recycled, or thrown away. Some pointers are given below:
```

Comment:
I think this section needs updating also, since it references secure deleting files on the actual storage medium which isn't really feasible now. There are too many layers of abstractions and SSDs don't provide an interface that can assure this.
So I think we should update this section to just be about erasing disks, which is still a thing :)